pipeval/common.py

```diff
@@ -0,0 +1,25 @@
+""" Common functions for PipeVal """
+import os
+
+# pylint: disable=C0103,W0613
+def skippedValidation(name):
+    """
+    Conditionally mark a validation to be skipped
+
+    If the environment variable f'PIPEVAL_SKIP_{name}' is set to `true`,
+    the decorated function will skip that validation.
+    """
+    def decorator(func):
```

Comment:
Consider using `functools.wraps`. https://stackoverflow.com/a/309000

Reply:
Is `functools.wraps` needed in the `decorator` function since it just return the same function without defining a new one?